BricklinkSharp.Client/BricklinkClient.cs

```diff
@@ -812,7 +813,7 @@ public void Dispose()
                   $"&incInstr={(includeInstructions ? "Y" : "N")}&incBox={(includeBox ? "Y" : "N")}&" +
                   $"incParts={(includeExtraParts ? "Y" : "N")}&breakSets={(breakSetsInSet ? "Y" : "N")}";
 
-        await MeasureRequestAsync(cancellationToken);
+        await MeasureRequestAsync(BricklinkApiResourceType.PartoutValue, HttpVerb.Get, null, cancellationToken);
```

Comment:
Do we need to call measure for this call? There is no rate limiting because it is basically a scraper...

Reply:
done